src/boot.tsx

```diff
@@ -66,7 +83,8 @@ export default (element: HTMLDivElement, hub: Hub) => {
     Object.assign(settings, {
       hint: getHint(),
       language: (element.getAttribute("data-lang") || "r") as Language,
-      pre_exercise_code: getText("pre-exercise-code"),
+      lang_version: element.getAttribute("lang-version"),
+      pre_exercise_code: getText("pre-exercise-code") + getPackages(element.getAttribute("packages")),
```

Comment:
Looks like you have duplication here

Reply:
Sorry David, not sure what you're referring to here.